joomla/joomla-cms
# Backport 2FA edit screen for com_admin

### Summary of Changes

Backport 2FA edit screen for com_admin from 4.x to 3.x too

### Testing Instructions

- enable atleast one 2fa plugin
- got to your profile in the backend
- notice that you can not setup 2fa there
- apply this patch (or install the packages from here: https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/30751/downloads/35668)
- got to your profile in the backend
- notice that you now can setup 2fa
- please setup 2fa
- logout
- login with 2fa
- remove 2fa
- logout
- login without 2fa

### Actual result BEFORE applying this Pull Request

It is not possible to configure 2fa from com_admin

### Expected result AFTER applying this Pull Request

We can configure 2fa from com_admin

### Documentation Changes Required

None
